src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -197,8 +193,11 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             }
         }
 
+        $path = $resourceMetadata->getAttribute('routePrefix', '');
```

Comment:
What do you think about sanitizing the prefix with `'/'.trim(trim($prefix, '/'))` if it's defined (`!== ''`)?

Reply:
I thought about that too but then my idea was that in Symfony, you always have to start with `/` both, for prefixes as well as routes. And if you define them incorrectly, you see an invalid path so that's a quickfix. Not sure we should do this :)

Reply:
My idea was just to follow the same logic as [`RouteCollection::addPrefix()`](https://github.com/symfony/symfony/blob/880d7e74360f42c8eaf7484e217093e3424c7b2c/src/Symfony/Component/Routing/RouteCollection.php#L143).
IMO it can improve the UX.

Reply:
Included 👍